apollo-router/src/plugins/connectors/handle_responses.rs

```
@@ -253,17 +247,7 @@ fn deserialize_response(
     {
         // Treat any JSON-y like content types as JSON
         // Also, because the HTTP spec says we should effectively "guess" the content type if there is no content type (None), we're going to guess it is JSON if the server has not specified one
-        serde_json::from_slice::<Value>(body).map_err(|_| {
-            if let Some(debug_context) = debug_context {
-                debug_context.lock().push_invalid_response(
-                    debug_request.0.clone(),
-                    parts,
-                    body,
-                    &connector.error_settings,
-                    debug_request.1.clone(),
-                );
-            }
-        })
+        serde_json::from_slice::<Value>(body).map_err(|_| ())
```

Why can we ignore the error here? Seems like it would indicate a serialization error?